<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MouseInfo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>Localização</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_local_mouse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>29</y>
     <width>381</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mov_mouse">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>55</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 65, 172);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string> ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_iniciar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>25</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 65, 172);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
